app.rb

```diff
@@ -1,9 +1,19 @@
 require 'sinatra/base'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/messageboard' do
+    @peep = Peep.all           
+    erb :messageboard
+  end
+
+  get '/chatter' do
```

Comment:
The naming here is confusing, if we said that the route about for all peeps should be **get /peeps** then this should be **get /peeps/new**